lib/pytis/util/resolver.py

```
@@ -84,16 +85,31 @@ def _import_module(self, name):
-                    # nested import within this module.
-                    raise ResolverError("Resolver error loading specification '%s': %s" % (name, e))
+                # Fall back to exception string matching for older Python versions.
+                if sys.version_info[0] == 2:
+                    matcher = re.compile("No module named (.*)")
+                else:
+                    matcher = re.compile("No module named '([^']*)'.*")
+                m = matcher.match(str(e))
+                if m:
+                    failed_import_name = m.group(1)
+                else:
+                    failed_import_name = None
```

Ruff (0.12.2)

Use ternary operator `failed_import_name = m.group(1) if m else None` instead of `if`-`else`-block

Replace `if`-`else`-block with `failed_import_name = m.group(1) if m else None`

(SIM108)